runtime/codert_vm/jswalk.c

```diff
@@ -186,7 +186,7 @@ UDATA  jitWalkStackFrames(J9StackWalkState *walkState)
 		walkState->outgoingArgCount = walkState->argCount;
 
 		if ((!(walkState->flags & J9_STACKWALK_SKIP_INLINES)) && getJitInlinedCallInfo(walkState->jitInfo)) {
-			jitGetMapsFromPC(walkState->walkThread->javaVM, walkState->jitInfo, (UDATA) walkState->pc, &(walkState->stackMap), &(walkState->inlineMap));
+			jitGetMapsFromPC(walkState->currentThread, walkState->jitInfo, (UDATA) walkState->pc, &(walkState->stackMap), &(walkState->inlineMap));
```

Comment:
I think this should continue to use `walkThread` which is the thread whose stack we are walking (in the context of `currentThread`).

Reply:
That's inconsistent with pretty much every other function. In the ASGCT context, the two threads are both necessarily the current thread.

Reply:
A bit more clarity - the ASGCT flag applies to the current thread. It is possible to observe it on another thread, and we'd be wrong to take action based on that.
```
Thread A halts thread B in order to walk its stack
<switch to B>
Thread B is interrupted and calls ASGCT setting the flag in B
<switch to A>
Thread A walks thread B stack
```
In this case using the walkThread to test the flag would be incorrect.

Reply:
I retract my concern; `jitGetMapsFromPC()` needs to know the context in which it is executing, not the thread whose PC is provided, so your changes are correct.